pennylane_lightning/core/src/algorithms/JacobianData.hpp

```diff
@@ -203,10 +203,10 @@ template <class StateVectorT> class OpsData {
  */
 template <class StateVectorT> class JacobianData {
   private:
-    using ComplexT = typename StateVectorT::ComplexT;
+    using CFP_t = typename StateVectorT::CFP_t;
```

Comment:
I thought we were going to replace these with ComplexT everywhere?
Can we make that change again?

Reply:
As discussed, I'll create a PR to refactor our numpy data type to cucomplex data type conversion pipeline to fix this issue.